features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -9,7 +9,7 @@ CREATE TABLE IDN_OAUTH_CONSUMER_APPS (
             CONSUMER_KEY VARCHAR (255) NOT NULL,
             CONSUMER_SECRET VARCHAR (2048),
             USERNAME VARCHAR (255),
-            TENANT_ID INTEGER DEFAULT 0,
+            TENANT_ID INTEGER DEFAULT 0 NOT NULL,
```

Comment:
is this change required?

Reply:
Yes. DB2 doesn't allow to define a unique key constraint over non null columns eventhough it's defined with a default value. Tested locally and previous schema didn't work. Hence had to add `NOT NULL`